lib/internal/process/coverage.js

```diff
@@ -9,7 +9,7 @@ const hasInspector = process.config.variables.v8_enable_inspector === 1 &&
 let session;
 
 function writeCoverage() {
-  if (!hasInspector || !session) {
+  if (!session) {
     return;
```

Comment:
The `session` check can not be reached due to either being set to a truthy value if the inspector is present, or not being set and the `setup` function throws and the code does not continue. If the latter is the case, the `writeCoverage` function is never called at all.

Therefore it's safe to remove this check as well.